src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -26,50 +26,63 @@
+            + "         " + COMMAND_WORD + " 1, 2, 3\n"
             + "         " + COMMAND_WORD + " " + PREFIX_TAG + "friend\n"
             + "         " + COMMAND_WORD + " " + PREFIX_TAG + "friend " + PREFIX_TAG + "enemy";
 
-    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
+    //need to check here
+    //public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
+    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person(s) Successful";
     public static final String MESSAGE_DELETE_TAG_SUCCESS = "Deleted Tags";
 
-    private final Index targetIndex;
+    private final ArrayList<Index> targetIndexes;
 
     private final Set<Tag> targetTags;
 
     public DeleteCommand(Index targetIndex) {
-        this.targetIndex = targetIndex;
+        this.targetIndexes = new ArrayList<>();
+        targetIndexes.add(targetIndex);
+        this.targetTags = null;
+    }
+
+    public DeleteCommand(ArrayList<Index> targetIndexes) {
+        this.targetIndexes = targetIndexes;
         this.targetTags = null;
     }
 
     public DeleteCommand(Set<Tag> targetTags) {
+        this.targetIndexes = null;
         this.targetTags = targetTags;
-        this.targetIndex = null;
     }
 
     @Override
     public CommandResult executeUndoableCommand() throws CommandException {
-        // this code block is command execution for delete [index]
-        if (targetTags == null && targetIndex != null) {
+        // this code block is command execution for delete [indexes]
+        if (targetTags == null && targetIndexes != null) {
             List<ReadOnlyPerson> lastShownList = model.getFilteredPersonList();
+            ArrayList<ReadOnlyPerson> deletePersonList = new ArrayList<>();
 
-            if (targetIndex.getZeroBased() >= lastShownList.size()) {
-                throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+            for (Index index : targetIndexes) {
+                if (index.getZeroBased() >= lastShownList.size()) {
+                    throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+                }
+                ReadOnlyPerson personToDelete = lastShownList.get(index.getZeroBased());
+                deletePersonList.add(personToDelete);
             }
 
-            ReadOnlyPerson personToDelete = lastShownList.get(targetIndex.getZeroBased());
-
             try {
-                model.deletePerson(personToDelete);
+                model.deletePersons(deletePersonList);
             } catch (PersonNotFoundException pnfe) {
-                assert false : "The target person cannot be missing";
+                assert false : "One of the target persons is missing";
             }
 
-            return new CommandResult(String.format(MESSAGE_DELETE_PERSON_SUCCESS, personToDelete));
+            return new CommandResult(String.format(MESSAGE_DELETE_PERSON_SUCCESS));
```

Comment:
wanna print out who are deleted?

Reply:
....I'll send another PR later this week to fix this problem!

Reply:
yes..i want ...

Reply:
haha oops sorry i didnt see ongoing haha